web/app/models.py

```diff
@@ -174,6 +174,8 @@ def to_dict(self):
 KOMI_VALUES = ['0', '0.5', '1', '1.5', '2', '2.5', '3.0', '3.5', '4', '4.5',
     '5', '5.5', '6', '6.5', '7', '7.5', '8', '8.5', '9', '9.5']
 TIE_BREAKS = ['SOS', 'SODOS', 'Face to Face Result', 'Random Procedure']
+MATCH_RESULTS = ['White Wins', 'Black Wins', 'Jigo', 'No Result',
```

Comment:
Is this from the tournament guide? 

Note: Most Go Game results are written as W+R, W+3.5, etc., -- White wins by resign, white wins by 3.5

Have you thought about how to the score differences?